WORKSPACE

```diff
@@ -67,6 +67,32 @@ yarn_install(
     yarn_lock = "//ui:yarn.lock",
 )
 
+# bats: Bash Automated Testing System
+http_archive(
```

Comment:
if you want to be able to re-use the `bats_test` rule from other repositories using enkit, the `http_archive` deps need to go in the `enkit_deps` macro, in `bazel/deps.bzl`

Reply:
Done.